ui/packages/catalog-ui-search/src/main/webapp/js/model/Workspace.collection.js

```diff
@@ -168,6 +147,21 @@ module.exports = Backbone.Collection.extend({
     )
     this.create(duplicateWorkspace)
   },
+  fetch() {
```

Comment:
@andrewkfiedler Is there a better place to do this...like when actually viewing a single workspace? And if so, where?

Reply:
This is fine for now, we can iterate on where we end up fetching.